app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Category Button -->
+    <dimen name="category_button_width">0dp</dimen>
```

Comment:
You don't have to put any `0dp` dimens in this file - just an Android convention.